src/ring_theory/ideal/basic.lean

```diff
@@ -77,6 +77,14 @@ theorem eq_top_iff_one : I = ⊤ ↔ (1:α) ∈ I :=
 theorem ne_top_iff_one : I ≠ ⊤ ↔ (1:α) ∉ I :=
 not_congr I.eq_top_iff_one
 
+theorem mem_iff_mul_unit_mem {x y : α} (hy : is_unit y) : x ∈ I ↔ y * x ∈ I :=
```

Comment:
If you swap the two sides of this iff, then it becomes a good simp-lemma.
About the name: the substring `mul_unit` indicates that you are multiplying the unit on the *right*. So what you want is `unit_mul`. Both versions are useful as simp-lemma.
After swapping the sides of the iff, I guess the names would be
`mul_unit_mem_iff` and `unit_mul_mem_iff`.